entry/src/entry.rs

```diff
@@ -111,15 +111,22 @@ pub struct Api<'a> {
 /// field points to Transactions that took place shortly before `hash` was generated.
 ///
 /// If you multiply `num_hashes` by the amount of time it takes to generate a new hash, you
-/// get a duration estimate since the last Entry. Since processing power increases
+/// get a duration estimate since the last `Entry`. Since processing power increases
 /// over time, one should expect the duration `num_hashes` represents to decrease proportionally.
 /// An upper bound on Duration can be estimated by assuming each hash was generated by the
 /// world's fastest processor at the time the entry was recorded. Or said another way, it
 /// is physically not possible for a shorter duration to have occurred if one assumes the
 /// hash was computed by the world's fastest processor at that time. The hash chain is both
 /// a Verifiable Delay Function (VDF) and a Proof of Work (not to be confused with Proof of
 /// Work consensus!)
-
+///
+/// `Entry` contains `transactions` that are executable in parallel. So an `Entry` is a unit of
```

Comment:
I think "solana core protocol" is the default context for most of the code in the validator.  I do not understand why you think this clarification is required.
I've applied your suggestion.
But it could be useful for me to understand why do you think it is better to say it this way.

Reply:
As there is nothing about the Entry struct itself or the way it is used in PoH that requires all transactions be parallelizable, the requirement is established by the TPU and the TVU in the way Entry is used. "Solana core protocol" was the general term I came up with to refer to TPU+TVU.
And fwiw, just because the solana validator is the only user of `solana-entry` now, that doesn't mean that will always be true.